.pre-commit-config.yaml

```diff
@@ -14,23 +14,24 @@ repos:
     rev: v4.6.0
     hooks:
       - id: check-yaml
-        exclude: "environment.yml"
+        exclude: '^{{ cookiecutter.repo_name }}/.*\.yml$' # {{ cookiecutter.repo_name }} contains jinja variables in .yml, hook fails
```

Comment:
customization needed for `pre-commit`, detailed comments provided for `exclude:`